client/app/components/application.coffee

```diff
@@ -139,18 +123,22 @@ module.exports = React.createClass
                                 isMailbox       : @state.isMailbox
                                 isLoading       : @state.isLoading
 
-                            if @state.isMailbox and @state.messageID
-                                Conversation
-                                    ref             : "conversation"
-                                    key             : "conversation-#{@state.messageID}"
-                                    messageID       : @state.messageID
-                                    conversationID  : message?.get 'conversationID'
-                                    subject         : message?.get 'subject'
-                                    conversation    : @state.conversation
-                            else
-                                section
-                                    'key'          : 'placeholder'
-                                    'aria-expanded': false
+                        if @state.isMailbox and @state.messageID
+                            Conversation
+                                ref             : "conversation"
+                                key             : "conversation-#{@state.messageID}"
+                                messageID       : @state.messageID
+                                conversationID  : message?.get 'conversationID'
+                                subject         : message?.get 'subject'
+                                conversation    : @state.conversation
+                        else
+                            section
+                                'key'          : 'placeholder'
+                                'aria-expanded': false
+
+            if @state.action is AccountActions.CREATE
+                div role: 'complementary',
+                    AccountWizardCreation()
```

Comment:
Not agree: we defines in architecture design sprint that _Application_ component and top-level components (such as _Menu_, _MailboxList_, _Conversation_, _Compose_, _Settings_…) manage their own states (for many reasons, like to prevent useless cascading re-renders). This wizard_ is just as the _Setting_ component: it's autonomous and can handle its own state.